fdbserver/storageserver.actor.cpp

```diff
@@ -8388,7 +8476,11 @@ class StorageUpdater {
 	  : currentVersion(fromVersion), fromVersion(fromVersion), restoredVersion(restoredVersion),
 	    processedStartKey(false), processedCacheStartKey(false) {}
 
-	void applyMutation(StorageServer* data, MutationRef const& m, Version ver, bool fromFetch) {
+	void applyMutation(StorageServer* data,
+	                   MutationRef const& m,
```

Comment:
a nit: do you think renaming variable from `encrypted` -> `encryptedMutation` (and maybe `m` -> `mutation` would improve code readability? :)

At some place `encrypted` is used as a boolean flag to pass if the encryption is supported and initially it got me bit confused.